### PR TITLE
BUG 1770709: Use "preserveUnknownFields: false" for the CRD

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -102,3 +102,13 @@ else
 endif
 
 verify: verify-gofmt
+
+sync-manifests:
```
Comment (zshi-redhat) [inline Makefile]: This may not work for 4.3 if there is API change in 4.4 development cycle. where only 4.4 needs to be updated.

Comment (pliurh) [reply Makefile]: Fiixed

**Pull request merged**